containers/ecr-viewer/src/styles/uswds-settings.scss

```diff
@@ -10,7 +10,6 @@
 
   // Component
   $theme-hero-image: "../../../../dist/img/hero.jpg",
-  $theme-accordion-button-background-color: "blue-cool-60",
```

Comment:
why remove this? did we purposefully change the accordion color?

Reply:
I changed it to use the same color("blue-cool-60") as everything else. Seemed weird to have just one bit done differently.